src/Material.cpp

```diff
@@ -414,9 +478,6 @@ void Material::setNumEnergyGroups(const int num_groups) {
 
     if (_chi != NULL)
       MM_FREE(_chi);
-
-    if (_fiss_matrix != NULL)
-      MM_FREE(_fiss_matrix);
```

Comment:
Simply to be consistent with the fact that `_fiss_matrix` is not deallocated below for non-aligned data. There is no need to deallocate `_fiss_matrix` here since the `Material::buildFissionMatrix()` will deal with deallocation / re-sizing of the array as needed.